actor/src/main/scala/org/apache/pekko/actor/AbstractActor.scala

```diff
@@ -363,6 +363,13 @@ abstract class UntypedAbstractActor extends AbstractActor {
  */
 abstract class AbstractLoggingActor extends AbstractActor with ActorLogging
 
+/**
+ * Java API: compatible with lambda expressions
+ *
+ * Actor base class that mixes in logging into the Actor.
+ */
```

Comment:
Can you put `@since 2.0.0` in Javadoc of the new classes?

Reply:
Thanks, seems like we now need both new methods and classes annotated.